Code/SimDivPickers/testPickers.cpp

```diff
@@ -37,19 +37,27 @@ void testGithub2245() {
   BOOST_LOG(rdErrorLog) << "Testing github issue 2245: MinMax Diversity picker "
                            "seeding shows deterministic / non-random behaviour."
                         << std::endl;
+  const int MAX_ALLOWED_FAILURES = 3;
+  int maxAllowedFailures;
   {
     RDPickers::MaxMinPicker pkr;
     int poolSz = 1000;
     auto picks1 = pkr.lazyPick(dist_on_line, poolSz, 10, RDKit::INT_VECT(), -1);
-    auto picks2 = pkr.lazyPick(dist_on_line, poolSz, 10, RDKit::INT_VECT(), -1);
-    TEST_ASSERT(picks1 != picks2);
+	for (maxAllowedFailures = MAX_ALLOWED_FAILURES; maxAllowedFailures; --maxAllowedFailures) {
```

Comment:
Rather than do a PR for a formatting change, I actually just pushed the changes directly.

Reply:
@greglandrum Oops, sorry, I hadn't realized! That's because I got a new laptop, I forgot to backup Notepad++ settings on the old one, and apparently I haven't restored all of them yet :-/ Thanks for taking care of reformatting those pesky tabs.